script/install-dependencies.sh

```diff
@@ -4,27 +4,30 @@ set -euo pipefail
 
 SUDO=sudo
 OS=$(lsb_release -cs)
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 if [ "$(id --user)" = 0 ]; then
   SUDO=
 fi
 
 # Install native package build dependencies.
-$SUDO apt-get install -y build-essential libudev-dev unzip git python pmount cups-client
+$SUDO apt-get install -y \
+  build-essential \
+  libudev-dev \
+  unzip \
+  git \
+  python \
+  pmount \
+  cups-client \
+  ruby
```

Comment:
where does the ruby come into play?

Reply:
`fpm`, used the build the `.deb`, is built with ruby.